lib/tasks/taxonomy.rake

```diff
@@ -2,13 +2,25 @@
 namespace :taxonomy do
   desc 'Change ownership of taxons from collections-publisher to content-tagger'
   task change_ownership: :environment do
-    ContentItem.transaction do
-      taxons = ContentItem.where(document_type: 'taxon')
+    ActiveRecord::Base.transaction do
+      taxons = ContentItem.where(document_type: 'taxon', publishing_app: "collections-publisher")
       puts "Found #{taxons.count} taxons"
+
       taxons.each do |taxon|
         puts "Changing #{taxon.title} from #{taxon.publishing_app} to content-tagger"
         taxon.update!(publishing_app: 'content-tagger')
       end
+
+      puts
```

Comment:
I think this would be nicer with an empty string just to indicate that you do intend to send an empty string as it looks a little like it is missing the argument.